src/omega_edit/server.ts

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode'
+import * as fs from 'fs'
+import * as unzip from 'unzip-stream'
+import * as os from 'os'
+import * as child_process from 'child_process'
+import { HttpClient } from 'typed-rest-client/HttpClient'
+import XDGAppPaths from 'xdg-app-paths'
+
+const xdgAppPaths = XDGAppPaths({ name: 'omega_edit' })
+
+class Backend {
+  constructor(readonly owner: string, readonly repo: string) {}
+}
+
+class Artifact {
+  constructor(readonly omegaEditVersion: string) {}
+
+  name = `omega-edit-scala-server-${this.omegaEditVersion}`
+  archive = `${this.name}.zip`
+  archiveUrl = (backend: Backend) =>
+    `https://github.com/${backend.owner}/${backend.repo}/releases/download/v${this.omegaEditVersion}/${this.archive}`
```

Comment:
Yes currently it does download it from the other repo. So I can make a ticket to fix this issue. As I believe if you are debugging locally it could download the zip and add it so that is available locally. There is a possibly I can also add a way the daffodil-vscode repo can build the omega-edit server packaging without having all the code in the repo so that getting the server works the same as getting the daffodil debugger server.